testing/component-testing-rtl/src/__tests__/2-using-react-testing-library.test.js

```diff
@@ -0,0 +1,176 @@
+/* eslint-disable no-unused-expressions */
+import React from 'react';
+import { render } from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
+import '../../test/setup';
+
+`
+⬇️
+
+You now know it can be tough to test nontrivial React components based
+on the elements they return.
+
+Your goal throughout these exercises is to leverage the power of React
+Testing Library to re-write our old tests into a series of smaller, more
+readable, and more robust new tests.
+
+Along the way, you will learn more about the React Testing Library API.
+We suggest you keep the React Testing Library docs open throughout this.
+
+⬆️
+`;
+
+`📚 Exercise one - Testing the Icon component with React Testing Library 📚
+
+In our first attempt, we tested that Icon renders an <img> and
```

Comment:
We changed this part of the test since we last talked, so this sentence should be adjusted (I don't think the previous exercise asserts that an `img` specifically is being rendered anymore.)

Reply:
I went back and looked, and we do actually test this in the _first_ exercise (the one that's common to both the Enzyme and RTL tests, where we test components without using either framework). I think it makes sense to keep it here... either that or remove it everywhere, but it might be useful to know.